frontend/src/pages/StudentProfile.vue

```diff
@@ -0,0 +1,17 @@
+<template>
```

Comment:
Potentially conflicting with PR request to create separate student profile page #10

Reply:
Also noticed the red border in the description class, could eventually be removed.

Reply:
I've deleted the StudentProfile.vue file!